python/amici/__init__.py

```diff
@@ -157,6 +157,8 @@ def ExpData(*args):
         # the constructor signature may have changed and you are glad this
         # wrapper did not break.
         return amici.ExpData(args[0].get(), *args[:1])
+    elif isinstance(args[0], ModelPtr):
+        return amici.ExpData(args[0].get())
```

Comment:
```suggestion
        return amici.ExpData(args[0].get(), *args[:1])
```

Reply:
For reasons I don't quite understand, this didn't work in 
4a13511. Keeping as was.